<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>1004</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bend Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="errLabel">
     <property name="styleSheet">
      <string notr="true">color: Red</string>
     </property>
     <property name="text">
      <string>Error</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="RefEditLayout">
     <item>
      <layout class="QHBoxLayout" name="addRemovBtns">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Edge Selection&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="buttonRefAdd">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Click button to enter selection mode,
click again to end selection</string>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="buttonRefRemove">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Click button to enter selection mode,
click again to end selection</string>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetReferences">
       <property name="toolTip">
        <string>- select an item to highlight it
- double-click on an item to see the fillets</string>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="batchSize">
        <number>80</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="basicOptionsLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Bend Parameters&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="invert">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Flips the direction of extrusion from the sketch plane&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Invert Bend Direction</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Bend Length:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Gui::QuantitySpinBox" name="length">
         <property name="toolTip">
          <string>Length to extrude the profile</string>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="unit" stdset="0">
          <string>mm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Bend Angle:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Gui::QuantitySpinBox" name="angle">
         <property name="toolTip">
          <string>Length to extrude the profile</string>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="unit" stdset="0">
          <string>deg</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Bend Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="BendType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Where to create material relative to the sketch profile&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="currentText">
          <string>Material Outside</string>
         </property>
         <item>
          <property name="text">
           <string>Material Outside</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Material Inside</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Thickness Outside</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Offset</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="offsetWidget">
       <item>
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Bend Offset:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Gui::QuantitySpinBox" name="offset">
         <property name="toolTip">
          <string>Internal radius of bend corners</string>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="unit" stdset="0">
          <string>mm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Bend Radius:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Gui::QuantitySpinBox" name="radius">
         <property name="toolTip">
          <string>Internal radius of bend corners</string>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="unit" stdset="0">
          <string>mm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Gap 1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Gui::QuantitySpinBox" name="gap1">
         <property name="toolTip">
          <string>Internal radius of bend corners</string>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="unit" stdset="0">
          <string>mm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Gap 2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Gui::QuantitySpinBox" name="gap2">
         <property name="toolTip">
          <string>Internal radius of bend corners</string>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="unit" stdset="0">
          <string>mm</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="showAdvanced">
     <property name="toolTip">
      <string>Show a live preview of the edited object</string>
     </property>
     <property name="text">
      <string>Show advanced options</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="advancedOptions" native="true">
     <layout class="QVBoxLayout" name="AdvancedOptionsLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="miterOptionsLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Miter Parameters&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="AutoMiter">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Flips the direction of extrusion from the sketch plane&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Auto Miter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Miter Angle 1:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Gui::QuantitySpinBox" name="miterangle1">
            <property name="toolTip">
             <string>Length to extrude the profile</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="unit" stdset="0">
             <string>deg</string>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Miter Angle 2:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Gui::QuantitySpinBox" name="miterangle2">
            <property name="toolTip">
             <string>Length to extrude the profile</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="unit" stdset="0">
             <string>deg</string>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Extend 1:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Gui::QuantitySpinBox" name="extend1">
            <property name="toolTip">
             <string>Internal radius of bend corners</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="unit" stdset="0">
             <string>mm</string>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Extend 2:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Gui::QuantitySpinBox" name="extend2">
            <property name="toolTip">
             <string>Internal radius of bend corners</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="unit" stdset="0">
             <string>mm</string>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>K Factor:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Gui::DoubleSpinBox" name="kfactor"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Max Ext Distance:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Gui::QuantitySpinBox" name="maxExtendDist">
            <property name="toolTip">
             <string>Length to extrude the profile</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="unit" stdset="0">
             <string>mm</string>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Min Gap:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Gui::QuantitySpinBox" name="minGap">
            <property name="toolTip">
             <string>Length to extrude the profile</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="unit" stdset="0">
             <string>mm</string>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="ReliefOptionsLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Relief Parameters&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="useReliefFactor">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Flips the direction of extrusion from the sketch plane&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Use Relief Factor</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Relief Type:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="reliefType">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Where to create material relative to the sketch profile&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="currentText">
             <string>Rectangle</string>
            </property>
            <item>
             <property name="text">
              <string>Rectangle</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Round</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Relief Factor:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Gui::DoubleSpinBox" name="ReliefFactor"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Minimum Relief Gap:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Gui::QuantitySpinBox" name="minReliefGap">
            <property name="toolTip">
             <string>Internal radius of bend corners</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="unit" stdset="0">
             <string>mm</string>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Relief Width:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Gui::QuantitySpinBox" name="reliefw">
            <property name="toolTip">
             <string>Internal radius of bend corners</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="unit" stdset="0">
             <string>mm</string>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Relief Depth:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Gui::QuantitySpinBox" name="reliefd">
            <property name="toolTip">
             <string>Internal radius of bend corners</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="unit" stdset="0">
             <string>mm</string>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="miscOptionslayout">
     <item>
      <widget class="QCheckBox" name="updateView">
       <property name="toolTip">
        <string>Show a live preview of the edited object</string>
       </property>
       <property name="text">
        <string>Update view</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="unfold">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Show a live preview of the edited object</string>
       </property>
       <property name="text">
        <string>Unfold</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>Gui/SpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
